lib/resource/index.js

```diff
@@ -39,9 +41,19 @@ var resourceAjaxRequest;
 var resourceDoing;
 var resourceAborting = false;
 
+function toggleSync(element) {
+  if (window.stopThatSync === false) {
+    window.stopThatSync = true;
+    element.innerHTML = "unfreeze position";
```

Comment:
Currently it is being done at the server side. It injects some localized strings into the script. See for example this snipped here:

`  string script =
  "var editorChapterLoaded = '" + locale_logic_text_loaded () + "';\n"
  "var editorChapterSaving = '" + locale_logic_text_saving () + "';\n"
  "var editorChapterSaved = '" + locale_logic_text_saved () + "';\n"
  "var editorChapterRetrying = '" + locale_logic_text_retrying () + "';\n"
  "var editorChapterReformat = '" + locale_logic_text_reformat () + "';\n"
  "var editorWriteAccess = true;\n"
  "var verticalCaretPosition = " + convert_to_string (verticalCaretPosition) + ";\n"
  "var unsentBibleDataTimeoutWarning = '" + bible_logic_unsent_unreceived_data_warning (false) + "';\n";
  config_logic_swipe_enabled (webserver_request, script);
  view.set_variable ("script", script);
`